ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -298,6 +298,15 @@
   @Default("1000")
   public final long routerOperationTrackerMinDataPointsRequired;
 
+  /**
```

Comment:
Did you run the code formatter on all the files?

Reply:
updated.